### ISSUE TITLE
Moving Project to Open Source?

### ISSUE THREAD

**Issue by beingmerry** [controlx-io/logic-prover-support#7]
I have been using your application for some testing and would like to make some changes to how the application works. Could you provide the application files as open source?

I plan to use this for a CODESYS project for testing.

**Comment by alex-controlx** [comment]
Yes it will be open source at some point. There were some changes in ElectronJS and it broke the build, so I need to fix it and  then push to a public repo. Stay tuned.

**Comment by CMor1184** [comment]
@alex-controlx, do you plan to open-source the project in the next days/weeks?

**Comment by alex-controlx** [comment]
good you reminded me. I should be able to do it next week unless something else urgent comes up.

The problem is, I updated all npm packages, and now I'm halfway in the rabbit hole fixing braking changes. I wish I just pushed it how it was, but it felt like the last thing to do 😄
